Tools/sitl_run.sh

```diff
@@ -79,6 +79,8 @@ if [ "$program" == "jmavsim" ] && [ ! -n "$no_sim" ]; then
 	SIM_PID=`echo $!`
 elif [ "$program" == "gazebo" ] && [ ! -n "$no_sim" ]; then
 	if [ -x "$(command -v gazebo)" ]; then
+		# Obtain the SDF filename of the model from the model.config file
+		model_file=`grep -oP "<sdf.*>(.*)</sdf>" "${src_path}"/Tools/sitl_gazebo/models/"${model}"/model.config | cut -d ">" -f 2 | cut -d "." -f 1`
```

Comment:
This handles generic names where a sdf file is specified in the model.config but I am not sure if the added complexity is worth it.

If all models are always going to be in the format of `modelname-gen.sdf` can we just handle it by checkinf if a `model-gen.sdf` exists?

Reply:
That's a good idea. I did not think of that. My mind always goes to the most general solution.

But I agree, let me implement that check for `modelname-gen.sdf`. That way, when someone is reading the code they will immediately know what is going on.